bin/generatePhosimInput.py

```diff
@@ -0,0 +1,129 @@
+                       brightestStar_gmag_inCat=11.0,
+                       brightestGal_gmag_inCat=11.0,
+                       sntable=sntable,
+                       sn_sedfile_prefix=os.path.join(sn_sed_file_dir, 'specFile_'))
+    # fname = phoSimInputFileName(obsHistID)
+    # if not os.path.exists(os.path.dirname(fname)):
+    #    os.makedirs(os.path.dirname(fname))
+    if not os.path.exists(sn_sed_file_dir):
+        os.makedirs(sn_sed_file_dir)
+    tSky.writePhoSimCatalog(fname)
+    availConns = tSky.availableConnections
+    tend = time.time()
+    print (obsHistID, tend - tstart)
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description='Write phoSim Instance Catalogs'
+                                     'and SN spectra to disk '
+                                     'example : '
+                                     'python generatePhosimInput.py 230\ '
+                                     '--OpSimDBDir ~/data/LSST/OpSimData/\ '
+                                     '--seddir "./"\ '
+                                     '--outfile phosim_instance_catalog_220.txt')
+    parser.add_argument('--opsimDB',
+                        type=str,
+                        help='OpSim database sqlite filename',
+                        default='minion_1016_sqlite.db')
+    parser.add_argument('visit',
+                        type=int,
+                        help='Visit number (obsHistID)')
+    parser.add_argument('--OpSimDBDir',
+                        help='absolute path to dir with the opsimBD',
+                        type=str,
+                        default='./')
+    parser.add_argument('--outfile', type=str, default=None,
+                        help='output filename for instance catalog')
+    parser.add_argument('--seddir',
+                        type=str,
+                        default=None,
+                        help='directory to contain SED files')
+    args = parser.parse_args()
+    # Set up OpSim database
+    opSimDBPath = os.path.join(args.OpSimDBDir, args.opsimDB)
+    engine = create_engine('sqlite:///' + opSimDBPath)
+
+    obs_gen = ObservationMetaDataGenerator(database=opSimDBPath)
+    sql_query = 'SELECT * FROM Summary WHERE ObsHistID == {}'.format(args.visit)
+    df = pd.read_sql_query(sql_query, engine)
+    recs = df.to_records()
+    obsMetaDataResults = obs_gen.ObservationMetaDataFromPointingArray(recs)
+    obs_metaData = obsMetaDataResults[0]
+    sn_sed_file_dir = os.path.join(args.seddir, 'spectra_files')
+
+    availConns = None
+    print('will generate pointing for {0} and write to filename {1}'.format(
+          obs_metaData._OpsimMetaData['obsHistID'], args.outfile))
+    generateSinglePointing(obs_metaData,
+                           availableConns=availConns,
+                           sntable='TwinkSN',
+                           fname=args.outfile,
+                           sn_sed_file_dir=sn_sed_file_dir)
```

Comment:
Does this work?  In the function declaration above none of the arguments to `generateSinglePointing` are keyword arguments.